modules/tinymce/src/core/test/ts/browser/content/EditorGetContentTextFormatTest.ts

```diff
@@ -10,10 +10,32 @@ describe('browser.tinymce.core.content.EditorGetContentTextFormatTest', () => {
     base_url: '/project/tinymce/js/tinymce'
   }, []);
 
-  it('get text format content should trim zwsp', () => {
+  const assertSelectedRadioButtons = (editor: Editor, nrOfInputs: number, shouldBeSelected: number) => {
+    const inputs = editor.getBody().getElementsByTagName('input');
+    let selected = 0;
+    for (let index = 0; index < inputs.length; index++) {
+      if (inputs[index].checked) {
+        selected++;
+      }
+    }
+
+    assert.equal(inputs.length, nrOfInputs, 'Should have the right amount of inputs');
+    assert.equal(selected, shouldBeSelected, 'Should have exactly one radio button');
+  };
```

Comment:
A little bit shorter:

```suggestion
  const assertSelectedRadioButtons = (editor: Editor, nrOfInputs: number, shouldBeSelected: number) => {
    const total = editor.getBody().querySelectorAll('input').length;
    const selected = editor.getBody().querySelectorAll('input:checked').length;

    assert.equal(total, nrOfInputs, 'Should have the right amount of inputs');
    assert.equal(selected, shouldBeSelected, 'Should have exactly one radio button');
  };
```